docs/libraries/azure-core/reference/decorators.md

```diff
@@ -22,7 +22,9 @@ Identifies a ModelProperty as containing the final location for the operation re
 
 #### Parameters
 
-None
+| Name        | Type                  | Description                                                                                                                                                 |
+| ----------- | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| finalResult | `union Model \| void` | Sets the expected return value for the final result. Overrides<br />any value provided in the decorated property, if the property uses ResourceLocation<T>. |
```

Comment:
Do we warn if there's a conflict?  Like, not a warning if there's an inferred value and they specify something that overrides it, but that they seemingly specific conflicting things (not sure that's possible)?

Reply:
No, this works the same way as the override for pollingOperation (which also overrides the link values), since it is an intentional override, and is largely used with templates (in the case of pollingLink, as an augment decorator)